Packs/FancyEmails/Scripts/MakeFancyEmailIndicatorTable/MakeFancyEmailIndicatorTable.py

```diff
@@ -0,0 +1,90 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+query = demisto.args().get('query')
+name = demisto.args().get('name')
+investigation_url = demisto.demistoUrls().get('investigation')
+use_alt_link = demisto.args().get('use_alt_link') in ['True', 'true']
+max_name_characters = int(demisto.args().get('max_name_characters', 40))
+
```

Comment:
Please insert this to the main function.
```suggestion
args = demisto.args()
query = args.get('query')
name = args.get('name')
investigation_url = demisto.demistoUrls().get('investigation')
use_alt_link = args.get('use_alt_link') in ['True', 'true']
max_name_characters = int(args.get('max_name_characters', 40))

```